integrations/salesforce-chat/src/script.raw.js

```diff
@@ -0,0 +1,71 @@
+(function () {
+    var EMBEDDED_SERVICE_SITE_URL = '<EMBEDDED_SERVICE_SITE_URL>';
+    var ORG_ID = '<ORG_ID>';
+    var MESSAGING_DEPLOYMENT_NAME = '<MESSAGING_DEPLOYMENT_NAME>';
+    var SCRT2_URL = '<SCRT2_URL>';
+
+    var w = window;
+    var d = document;
+
+    function loadScript(src, onload) {
+        var s = d.createElement('script');
+        s.type = 'text/javascript';
+        s.async = true;
+        s.src = src;
+        s.onload = onload;
+        var x = d.getElementsByTagName('script')[0];
+        x.parentNode.insertBefore(s, x);
+    }
+
+    function initMessaging() {
+        if (!w.embeddedservice_bootstrap) {
+            return;
+        }
+
+        w.embeddedservice_bootstrap.settings.language = 'en_US';
+
+        try {
+            w.embeddedservice_bootstrap.init(
+                ORG_ID,
+                MESSAGING_DEPLOYMENT_NAME,
+                EMBEDDED_SERVICE_SITE_URL,
+                {
+                    scrt2URL: SCRT2_URL,
+                },
+            );
+        } catch (err) {
+            // initialization failed
+        }
```

Comment:
Looking at other integrations, seems like we never log nor report these kind of errors, so I left it like this for now